question-generation/requirements.txt

```diff
@@ -0,0 +1,5 @@
+--find-links https://download.pytorch.org/whl/cu118
```

Comment:
Should this actually be in `requirements`?

Reply:
May remove `requirements.txt` in favor of conda's env yml.